index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en-GB" >
+<head>
+	
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Damian HOW</title>
+	<link rel="stylesheet" href="styles.css" type="text/css" />
+	<link rel="stylesheet" href="normalize.css" type="text/css" />
+	<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono&display=swap" rel="stylesheet" />
+	
+</head>
+
+<body>
+	<div id="page-container" >
+		<div id="content-wrap" >
```

Comment:
can remove the content wrap id and remove the bottom margin on the main tag to acheive the same effect